Common/Brokerages/InteractiveBrokersBrokerageModel.cs

```diff
@@ -75,6 +75,46 @@ public override bool CanSubmitOrder(Security security, Order order, out Brokerag
                 return false;
             }
 
+            // validate guaranteed VWAP order
+            if (order.Type == OrderType.Vwap)
+            {
+                // only supported with equities
+                if (security.Type != SecurityType.Equity)
+                {
+                    message = new BrokerageMessageEvent(BrokerageMessageType.Warning, "NotSupported",
+                        "This model does not support " + order.Type + " order type with " + security.Type + " security type."
+                    );
+
+                    return false;
+                }
+
+                // can only be submitted at or before 9.29 AM
+                var orderTime = order.Time.ConvertFromUtc(security.Exchange.TimeZone);
+                if (orderTime > orderTime.Date.Add(new TimeSpan(9, 29, 0)))
```

Comment:
this is probably the correct place for the time checks (vs `PreOrderChecksImpl`), I imagine different brokers may have different requirements since it's an in-house service they provide.

Reply:
👍 I'll update the cut-off time from 9.29 to 9.28